<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camerasetupdialog</class>
 <widget class="QDialog" name="camerasetupdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search cameras</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cameraList">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera framerate</string>
   </property>
  </widget>
  <widget class="QSlider" name="fpsSlider">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>65</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>15</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="singleStep">
    <number>15</number>
   </property>
   <property name="pageStep">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelFPS">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fpsList">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="formatList">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>camerasetupdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>camerasetupdialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
